### PR TITLE
RHS init doc.

### PR DESCRIPTION
**Pull request by demonsh**

### CODE REVIEW

File: mkdocs/docs/services/rhs.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+![](https://i.imgur.com/3ZS1ZvJ.png)
+> Identity State Diagram for Direct Identity
+
+The public data of identity includes:
+
+- The identity state, which is stored on the blockchain.
+- Roots of the identity trees: Claims tree(ClT), Revocation tree(ReT), and Roots tree(RoT).
+- All the nodes of the Revocation tree and Roots tree are public.
+
+The Reverse Hash Service stores nodes of SMTs (In our case, the CIT / ReT / RoT and identity state. This service enables users to perform reverse hash lookups and construct proofs of `existence` or `non-existence` of elements without revealing the specific element being proved. The main objective of this service is to enhance the privacy and security of iden3 identities while ensuring data integrity.
```
Comment (vladyslavmunin) [inline mkdocs/docs/services/rhs.md:20]:
```suggestion
The Reverse Hash Service stores nodes of SMTs (In our case, the CIT / ReT / RoT and identity state). This service enables users to perform reverse hash lookups and construct proofs of `existence` or `non-existence` of elements without revealing the specific element being proved. The main objective of this service is to enhance the privacy and security of iden3 identities while ensuring data integrity.
```

File: mkdocs/docs/services/rhs.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+### Credential Status
+
+If identity is published to RHS then we can check the credential status by doing a reverse hash lookup for the credential revocation nonce. In this case credential issued by this identity may have credential status type `Iden3ReverseSparseMerkleTreeProof`
+
+```json
+...{    
+"type": "Iden3ReverseSparseMerkleTreeProof"    
+"id": "https://rhs-staging.polygonid.me/node",
```
Comment (vladyslavmunin) [inline mkdocs/docs/services/rhs.md:140]:
```suggestion
"id": "https://rhs-staging.polygonid.me",    
```

File: mkdocs/docs/services/rhs.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+[https://github.com/iden3/merkletree-proof](https://github.com/iden3/merkletree-proof)
+
+### Links
+
+[https://docs.iden3.io/publications/pdfs/Merkle-Tree.pdf](https://docs.iden3.io/publications/pdfs/Merkle-Tree.pdf)
+
+[https://github.com/iden3/go-merkletree-sql](https://github.com/iden3/go-merkletree-sql)
+
+[https://github.com/0xPolygonID/js-sdk](https://github.com/0xPolygonID/js-sdk)
```
Comment (vladyslavmunin) [inline mkdocs/docs/services/rhs.md:172]:
```suggestion
[https://github.com/0xPolygonID/js-sdk/blob/2f711aaa45126bf16781d4daa4fe2f166470ce41/src/credentials/revocation.ts](https://github.com/0xPolygonID/js-sdk)
```

Comment (demonsh) [reply mkdocs/docs/services/rhs.md:172]: I think better to leave a link to the SDK itself.

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**